webui/react/src/pages/DesignKit.tsx

```diff
@@ -106,6 +108,88 @@ const ComponentSection: React.FC<Props> = ({ children, id, title }: Props): JSX.
   );
 };
 
+const PopoverSection: React.FC = () => {
+  const text = 'Popover text';
+  const buttonWidth = 70;
+  return (
+    <ComponentSection id="Popover" title="Popover">
+      <AntDCard>
+        <p>
+          <code>{'<Popover>'}</code> is a simple popup menu to provide extra information or
+          operations.
+        </p>
+      </AntDCard>
+      <AntDCard title="Best practices">
+        <p>
+          Comparing with <code>{'<Tooltip>'}</code>, besides information <code>{'<Popover>'}</code>{' '}
+          card can also provide action elements like links and buttons.
+        </p>
+      </AntDCard>
+      <AntDCard title="Usage">
+        <strong>Popover default</strong>
+        <p>
+          <Popover content={text}>
+            <code>{'<Popover>'}</code>
+          </Popover>
+        </p>
+        <strong>Popover placement</strong>
+        <div>
+          <div style={{ marginLeft: buttonWidth, whiteSpace: 'nowrap' }}>
+            <Popover content={text} placement="topLeft">
+              <Button>TL</Button>
+            </Popover>
+            <Popover content={text} placement="top">
+              <Button>Top</Button>
+            </Popover>
+            <Popover content={text} placement="topRight">
+              <Button>TR</Button>
+            </Popover>
+          </div>
+          <div style={{ float: 'left', width: buttonWidth }}>
+            <Popover content={text} placement="leftTop">
+              <Button>LT</Button>
+            </Popover>
+            <Popover content={text} placement="left">
+              <Button>Left</Button>
+            </Popover>
+            <Popover content={text} placement="leftBottom">
+              <Button>LB</Button>
+            </Popover>
+          </div>
+          <div style={{ marginLeft: buttonWidth * 4 + 24, width: buttonWidth }}>
+            <Popover content={text} placement="rightTop">
+              <Button>RT</Button>
+            </Popover>
+            <Popover content={text} placement="right">
+              <Button>Right</Button>
+            </Popover>
+            <Popover content={text} placement="rightBottom">
+              <Button>RB</Button>
+            </Popover>
+          </div>
+          <div style={{ clear: 'both', marginLeft: buttonWidth, whiteSpace: 'nowrap' }}>
```

Comment:
looks like these bottom-positioned ones have different padding, and background color in dark mode, from the top/right/left ones.

also think maybe we should actually use the grey background color in dark mode instead of the black (more contrast, matches Tooltip colors).